### PR TITLE
Command is not correct

### PR DESCRIPTION
**Pull request by ms-ishiyam**
We can use this command for secret not certificates because this command is not correct for certificates of KeyVault. When we use this command, the id will be below. I think this id is not correct for AppGW. 
https://xxxxx.vault.azure.net:443/certificates/xxxx/

The should be below. 

"KeyVaultSecretId": "https://xxxx.vault.azure.net:443/secrets/xxxx/"

When I used below command, I encountered error.

$secretId = $secret.Id.Replace($secret.Version, "")

"Set-AzApplicationGateway : SecretId 'https://xxxx.vault.azure.net:443/certificates/selftest/' specified in '/subscriptions/xxxxxxxxx/resourceGroups/apgw/p
roviders/Microsoft.Network/applicationGateways/AppGW/sslCertificates/SelfKeyCert' is invalid."

When I used below command, the operation was successful. So I think this document is not correct.

$secretId = $certificate.SecretId.Replace($certificate.Version, "")

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/Network/Network/help/Add-AzApplicationGatewaySslCertificate.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ This command gets an application gateway named ApplicationGateway01 and then add
 ```
 PS C:\> $AppGW = Get-AzApplicationGateway -Name "ApplicationGateway01" -ResourceGroupName "ResourceGroup01"
 PS C:\> $secret = Get-AzKeyVaultCertificate -VaultName "keyvault01" -Name "sslCert01"
-PS C:\> $secretId = $secret.Id.Replace($secret.Version, "") # https://<keyvaultname>.vault.azure.net/secrets/
+PS C:\> $secretId = $certificate.SecretId.Replace($certificate.Version, "") # https://<keyvaultname>.vault.azure.net/secrets/
```
Comment (VeryEarly) [inline src/Network/Network/help/Add-AzApplicationGatewaySslCertificate.md:40]: can you point me where "$certificate" was defined?

**Comment by hisasima** [comment]
Sorry, I didn't write some commands.

$certificate = Get-AzKeyVaultCertificate -VaultName $kv -Name "cert1" $secretId = $certificate.SecretId.Replace($certificate.Version, "")

I got it from below document.

https://docs.microsoft.com/en-us/azure/application-gateway/configure-keyvault-ps

**Pull request merged**